camel-route-support/src/main/java/io/kaoto/backend/api/service/deployment/generator/camelroute/IntegrationDeploymentGeneratorService.java

```diff
@@ -77,12 +77,7 @@ public String parse(List<StepParserService.ParseResult<Step>> flows) {
     public CustomResource parse(final String input) {
         if (stepParserService.appliesTo(input)) {
             try {
-                ObjectMapper yamlMapper =
-                        new ObjectMapper(new YAMLFactory())
-                                .configure(
-                                        DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES,
```

Comment:
What are the implications of failing now?
if an attribute is removed for instance in next version of Camel and file not updated yet.

Reply:
Before if an attribute was not recognized, it was silently ignored.

Now it will throw an error.

This comes I guess with the decision on what to do with legacy Camel CRDs. Do we want to keep them?